api-spec.md

```diff
@@ -287,3 +419,83 @@ All Extensions **should** use attribute names qualified from the root of Item, r
+| STAC Response             | STAC | <http://stacspec.org/spec/api/1.0.0-beta.1/req/stac-response>    | Specifies that OAFeat's relevant responses conform to STAC: STAC Collections for OGC API Commons - [collections](http://docs.opengeospatial.org/DRAFTS/20-024.html#rc_collections-section) & STAC Items for OGC API's [Features](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#_items_) Collections from the OAFeat `/collections` and `/collections/{collectionId}` endpoints.            |
+| OAFeat Core               | OAFeat | <http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/core>    | The core OGC API - Features endpoints & parameters Returns one or more STAC Collections from the OAFeat `/collections` and `/collections/{collectionId}` endpoints. Depends on OAFeat Core.         ([reference](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#rc_core))                         |
+| OpenAPI specification 3.0 | OAFeat | <http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/oas30>   | Describes the API as OpenAPI 3.0 ([reference](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#rc_oas30))                                          |
+| GeoJSON                   | OAFeat | <http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/geojson> | Requires OGC API - Features responses to be in GeoJSON ([reference](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#_requirements_class_geojson)) |                                             |
+
+Additional conformance classes are specified in the STAC Extensions.
+
+TODO: I'll fold these back into our main definitions above. For now I'm just aiming to get them fleshed out.
+
+### STAC Search (old)
+
+TODO: This is mostly specified above. Should figure out the revamp that talks about them in the right place.
+
+STAC Search is the conformance class for the core search endpoint that has the `rel` of `search`. It does not require
+OAFeat to be implemented, but its query parameters are mostly the same, with some sensible additions like being able to specify
+which 'collections' to query. 
+
+It is required to implement GET, and POST is optional. 
+
+### Commons
+
+[OGC API - Common - Part 2: Geospatial Data](http://docs.opengeospatial.org/DRAFTS/20-024.html) specifies a [Collections requirements 
+class](http://docs.opengeospatial.org/DRAFTS/20-024.html#rc_collections-section) that describes how to request information on Collections
+from an API. This provides the `data` rel type, that links to a list of collections that can all be referred to by ID. It is most often used
+in conjunction with STAC Response, to return valid STAC Collections.
+
+### STAC Response
+
+The STAC Response conformance class specifies that returned OGC Collections should be valid STAC Collections, and that returned OGC Features
```

Comment:
Shouldn't it also specify that returned catalogs are valid STAC catalogs?

Reply:
I can call that out, but a STAC Collection *is* a STAC Catalog, so that's probably why I left it out...

Reply:
In that case it should say "OGC Catalogs" because that would include Catalogs and Collections, but saying "OGC Collections" indicates it doesn't include Catalogs that aren't Collections.

Reply:
There is no "OGC Catalog" defined AFAIK.